scripts/js/commands/api/updateApiDocs.ts

```diff
@@ -92,6 +92,21 @@ export async function generateVersion(
   console.log(`Run pipeline for ${pkg.name}:${pkg.versionWithoutPatch}`);
   await runConversionPipeline(sphinxArtifactFolder, "docs", "public/docs", pkg);
   await generateHistoricalRedirects();
+
+  if (pkg.name === "qiskit") {
+    const majorVersion = Number(pkg.version.split(".").pop());
+    if (majorVersion >= 2) {
+      await generateVersion(
+        await Pkg.fromArgs(
+          "qiskit-c",
+          pkg.version,
+          pkg.versionWithoutPatch,
+          pkg.type,
+        ),
+        args,
+      );
+    }
+  }
```

Comment:
This might be annoying when generating exclusively Python when you're iterating on Python. But I'm unsure, I can see both sides. I'm curious what @arnaucasau thinks is the best workflow?

Reply:
If it makes a difference, the C docs are very quick to generate so it shouldn't be too annoying

Reply:
Sgtm. This PR looks good then other than the README

Reply:
While writing that up, I realised I'm pushing another thing to remember on to future us, and it shouldn't be too hard to have the C API use the qiskit entry in the json file. Gonna convert to draft while I test it out.

Reply:
Yeah, I agree with Arnau. My ideal would be:

1. Keep your unification of the HTML artifacts file. That's great!
2. Manually trigger creation of both packages

Reply:
Thanks for the feedback, done in 588dd806efc725e4137351fa0a45b2e2246a7e8b